src/FixedPointNumbers.jl

```diff
@@ -625,20 +624,8 @@ if VERSION >= v"1.1" # work around https://github.com/JuliaLang/julia/issues/341
     _precompile_()
 end
 
-@static if VERSION >= v"1.9.0-DEV.873"
-    function __init__()
-        # Statistics and SparseArrays are moved out from sysimage
-        # https://github.com/JuliaLang/julia/pull/44247#issuecomment-1172847231
-        @require Statistics="10745b16-79ce-11e8-11f9-7d13ad32a3b2" begin
-            Statistics._mean_promote(x::Real, y::FixedPoint) = Treduce(y)
-        end
-    end
-else
-    import Statistics
-    if isdefined(Statistics, :_mean_promote)
-        # https://github.com/JuliaMath/FixedPointNumbers.jl/pull/183
-        Statistics._mean_promote(x::Real, y::FixedPoint) = Treduce(y)
-    end
+if !isdefined(Base, :get_extension)
+    include("../ext/FixedPointNumbersStatisticsExt.jl")
```

Comment:
Although this is in keeping with PR #255 manner, I would prefer that the `precompile` be at the end of the module (except for the `__init__()`).

Reply:
Fixed in https://github.com/JuliaMath/FixedPointNumbers.jl/pull/277/commits/c4eee40f2aa33258e150259de49468bf73c20838.

Reply:
Actually, I'm not familiar with `_precompile_()` function. Perhaps the current code should be replaced with [PrecompileTools.jl](https://github.com/JuliaLang/PrecompileTools.jl).

Reply:
Yes, it is from the SnoopCompile.jl days.
I think replacing before releasing the new version is a good idea.